client/my-sites/stats/stats-post-anniversaries/index.jsx

```diff
@@ -0,0 +1,209 @@
+/*
+ * Post Anniversaries
+ *
+ * This component displays the post anniversaries based on the current day.
+ *
+ * There is no loading or empty state: the module is only displayed if there is
+ * at least one anniversary. That is the reason why this module should ideally
+ * be inserted at the end of a column.
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize, moment } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import StatsList from '../stats-list';
+import StatsListLegend from '../stats-list/legend';
+import SectionHeader from 'components/section-header';
+import StatsContentText from '../stats-module/content-text';
+import Card from 'components/card';
+import QueryPosts from 'components/data/query-posts';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSitePostsForQuery } from 'state/posts/selectors';
+
+// Maximum number of simultaneous queries to fetch the posts by year
+const MAX_POST_QUERIES = 10;
+
+const StatModulePostAnniversaries = props => {
+	const { oldestPostQuery, postsByYearQueries, postsByYear, siteId, translate } = props;
+
+	if ( ! siteId ) {
+		return null;
+	}
+
+	const allPosts = Array.prototype.concat( ...postsByYear );
+
+	return (
+		<div>
+			{ <QueryPosts siteId={ siteId } query={ oldestPostQuery } /> }
+			{ postsByYearQueries.map(
+				( query, i ) => <QueryPosts key={ i } siteId={ siteId } query={ query } />,
+			) }
+			{ ! allPosts.length
+				? null
+				: <div>
+						<SectionHeader
+							label={ translate(
+								'Anniversaries',
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Happy Anniversary!' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).